corehq/apps/users/bulk_download.py

```diff
@@ -129,7 +132,6 @@ def get_user_role_name(domain_membership):
 
 
 def make_web_user_dict(user, location_cache, domain):
-    user = CouchUser.wrap_correctly(user['doc'])
```

Comment:
So this is not needed now because its already wrapped correctly before calling this method?

Reply:
yep, the `wrap_correctly` call at the end of `_get_users_by_filters` takes care of it